src/services/EdFi.DataManagementService.Api/Core/Exceptions/FailureResponse.cs

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Text.Json.Serialization;
+
+namespace EdFi.DataManagementService.Api.Core.Exceptions;
+
+public record FailureResponse()
```

Comment:
Add summary documentation on the purpose of this, and referring to https://techdocs.ed-fi.org/pages/viewpage.action?spaceKey=EDFIODS&title=Error+Response+Updates